.env.example

```
@@ -4,6 +4,8 @@ POSTGRES_HOST=db
 POSTGRES_DB=care
 POSTGRES_PORT=5432
 DATABASE_URL=postgres://postgres:postgres@localhost:5433/care
+BACKUP_DIR="/home/$USER/care-backups"
+RETENTION_PERIOD=7
```

```suggestion
DB_BACKUP_RETENTION_PERIOD=7
```

replace in other places